content/css/concepts/display/display.md

```diff
@@ -46,6 +48,154 @@ The following values can be appplied to the `display` property:
+
+```css
+.block-element {
+  display: block;
+}
+```
+
+### 2. Display: inline
+
+Elements with `display: inline;` are rendered as inline elements. They don't create line breaks and occupy only the necessary width and height based on their content.
+
+```css
+.inline-element {
+  display: inline;
+}
+```
+
+### 3. Display: inline-block
+
+Elements with `display: inline-block;` are rendered as inline-level block elements. They behave like inline elements but allow you to set width, height, padding, and margins.
+
+```css
+.inline-block-element {
+  display: inline-block;
+}
+```
+
+### 4. Display: none
+
+Elements with `display: none;` are not rendered and are completely hidden from the page. They don't take up any space in the document flow.
+
+```css
+.hidden-element {
+  display: none;
+}
+```
+
+### 5. Display: flex
+
+Elements with `display: flex;` create a flex container, allowing you to build flexible and responsive layouts. Flex items inside the container can be positioned and resized using flexbox properties.
+
+```css
+.flex-container {
+  display: flex;
+}
+```
+
+### 6. Display: grid
+
+Elements with `display: grid;` create a grid container that enables you to define rows and columns, creating a two-dimensional grid system. Child elements inside the container can be placed in specific cells and adjusted for size and alignment.
+```css
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 10px;
+}
+```
+
+## Overriding display values
+
+CSS specificity allows you to override default display values of elements. Here's an example of overriding display:
+
+```css
+/* Original CSS */
+.element {
+  display: inline;
+}
+
+/* Overriding with higher specificity */
+.container .element {
+  display: block;
+}
+```
+
+## Responsive design considerations
+
+The display property can be combined with media queries to create responsive designs. Here's an example:
+
+```css
+@media (max-width: 768px) {
+  .element {
+    display: none; /* Hide element on smaller screens */
+  }
+}
+
+@media (min-width: 768px) {
+  .element {
+    display: block; /* Display element on larger screens */
+  }
+}
+```
+
+## `display: grid` and `display: flex` Comparison
+
+CSS Grid and Flexbox are powerful layout tools. The choice between display: flex and display: grid depends on the layout requirements and the desired design approach. 
+Flexbox is particularly useful for creating responsive and flexible layouts, especially when dealing with content in a linear fashion, such as navigation menus, image galleries, or vertically aligned elements. 
+CSS Grid is well-suited for creating advanced and flexible layouts, such as magazine-style designs, multi-column layouts, or designs that require explicit control over the placement of elements in a grid-like structure. Here is an example illustrating the difference between the two:
+
+```css
```

Comment:
For the tabbing in this example use 2 spaces instead of using tabs. So all the lines in the example will shift to the left after this is done.